transaction/src/plan/memo.rs

```diff
@@ -9,20 +9,30 @@ use rand::{CryptoRng, RngCore};
 
 #[derive(Clone, Debug)]
 pub struct MemoPlan {
-    pub plaintext: MemoPlaintext,
+    pub plaintext: [u8; MEMO_LEN_BYTES],
```

Comment:
The `MemoPlan` is meant to be human-readable, which means that, I think, this should be a `String` here.